### PR TITLE
Credential details window and configurations

### PR DESCRIPTION
**Pull request by jameseaster**
#### _Resolves - add credential details window and credential UI configurability_

---

### What kind of change does this PR introduce?

- UI update.

<br/>

### What is the current behavior?

- Delete button does not have a confirmation prompt.
- Credential cards are not configurable in style, overrides, or detail highlights.

<br/>

### What is the new behavior?

- Updated, mobile friendly details window for credential.
- Confirmation dialog appears when deleting a credential.
- JSON configurations to customize credential styles, overrides, & highlights.
- Update search bar functionality to consider credential overrides.

<br/>

### Does this PR introduce a breaking change?

- [ ] Yes
- [X] No

<br/>

### How has this been tested?

- Locally rendered

<br/>

### Screenshots: n/a

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,14 @@
 # bedrock-vue-wallet ChangeLog
 
+## 27.1.0 - 2024-mm-dd
```
Comment (dlongley) [inline CHANGELOG.md:3]:
```suggestion
## 28.0.0 - 2024-mm-dd
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,14 @@
...
+
+### Add
+- New details window for credential.
+- Delete credential confirmation dialog.
+- Full screen modal for small devices.
```
Comment (dlongley) [inline CHANGELOG.md:8]:
```suggestion
- **BREAKING**: New details window for credential.
- **BREAKING**: Delete credential confirmation dialog.
- **BREAKING**: Full screen modal for small devices.
```

File: components/CredentialDashboard.vue [hunk 3]
```diff
@@ -26,9 +26,14 @@
  * Copyright (c) 2018-2022 Digital Bazaar, Inc. All rights reserved.
```
Comment (dlongley) [inline components/CredentialDashboard.vue:26]:
```suggestion
 * Copyright (c) 2018-2024 Digital Bazaar, Inc. All rights reserved.
```

File: components/CredentialDashboard.vue [hunk 4]
```diff
@@ -26,9 +26,14 @@
...
  */
 import {computed, ref, toRef, watch} from 'vue';
+import {formatString, getValueFromPointer} from '../lib/helpers.js';
 import {createEmitExtendable} from '@digitalbazaar/vue-extendable-event';
 import CredentialsList from './CredentialsList.vue';
 import SearchBox from './SearchBox.vue';
+import vcViewConfigurations from '../lib/vcViewConfigurations.json';
```
Comment (dlongley) [inline components/CredentialDashboard.vue:33]: This should instead be incorporated into the main frontend config somewhere in this module's config namespace.

**Review by dlongley — CHANGES_REQUESTED** [review]
The biggest change needed here is to move the new config stuff into the main, shared, application frontend config under this module's namespace.

File: components/CredentialDashboard.vue (hunk 4 above)
Comment (jameseaster) [reply components/CredentialDashboard.vue]: Updated with commit 400b31e and 7b34a3d

File: lib/config.js [hunk 5]
```diff
@@ -2,8 +2,11 @@
  * Copyright (c) 2015-2022 Digital Bazaar, Inc. All rights reserved.
  */
 import {config} from '@bedrock/web';
+import {vcViewConfigurations} from './vcViewConfigurations.js';
 
 const cfg = {
+  // Customizable VC configurations
+  vcViewConfigurations,
```
Comment (dlongley) [inline lib/config.js:9]: So using the name `configurations` inside of the config is redundant now -- we can shorten it. Will take this to another medium for discussion.

Comment (jameseaster) [reply lib/config.js]: Rename vcViewConfigurations to cardDesigns - 7b2911f

**Review by dlongley — APPROVED** [review]

**Pull request merged**